### PR TITLE
HomeTV App: add more meals

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1342
[Problem] Add breakfast widget to app
[Solution]
    - added section changer
    - images download from:
            https://commons.wikimedia.org/wiki/File:Vegetarian_breakfast_at_Cafe_Cuilles.jpg
            https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Korean.cuisine-Bibimbap-01.jpg/800px-Korean.cuisine-Bibimbap-01.jpg

### CODE REVIEW

**Comment by lslachciak** [comment]
I think that bottom area "Hotel Restaurant App" is not needed at this moment, 
@singa2000 how do you think?
Anyway if we decide to stay with it, it should be constant i.e. don't slide together with other elements.

File: examples/mobile/HomeApp/index.html [hunk 1]
```diff
@@ -45,6 +45,43 @@
...
+							<div class="ui-li-text">
+								<span class="ui-li-text-title">
+									English Breakfast
+								</span>
+								<span class="ui-li-text-sub">
+									Delicious breakfast: fried egg, baked bacon, beans, sausages, tomato, bread, hot drink.
+								</span>
+							</div>
+							<div class="app-hotel-order">
+								<a href="#" class="ui-btn ui-btn-contained-colored" data-inline="true">Order</a>
```
Comment (lslachciak) [inline examples/mobile/HomeApp/index.html]: can we point to breakfast-menu.html
so we will be compatible with https://github.com/Samsung/TAU/pull/1346/

**Review by korneliakobiela — APPROVED** [review]
LGTM

**Comment by singa2000** [comment]
@TomaszLukawskiSam 
Would you check the conflicting files, please?

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Comment by TomaszLukawskiSam** [comment]
> @TomaszLukawskiSam
> Would you check the conflicting files, please?

Hi, I solved the merge conflict.

**Pull request merged**